src/main/java/com/microsoft/sqlserver/jdbc/PersistentTokenCacheAccessAspect.java

```diff
@@ -22,13 +22,16 @@
  * @see <a href="https://aka.ms/msal4j-token-cache">https://aka.ms/msal4j-token-cache</a>
  */
 public class PersistentTokenCacheAccessAspect implements ITokenCacheAccessAspect {
-    private static PersistentTokenCacheAccessAspect instance = new PersistentTokenCacheAccessAspect();
-
+    private static PersistentTokenCacheAccessAspect instance;
     private final Lock lock = new ReentrantLock();
 
-    private PersistentTokenCacheAccessAspect() {}
+    static final long TIME_TO_LIVE = 86400000L; // Token cache time to live (24 hrs).
```

Comment:
"Magic Number". Please use a numeric constant variable with a descriptive name instead of a numeric literal. You should never need to add a comment to document the value/meaning/purpose of a numeric literal. If this is done for all numeric values it will make the code more self documenting and simplify future maintenance efforts

Reply:
`TIME_TO_LIVE` is the numeric constant. I can be more descriptive in the name eg. maybe `TWENTY_FOUR_...` but that gets long and potentially `TIME_TO_LIVE`'s value can change. So, a compromise would be to use a short name and a comment indicating the time in hours (b/c 86400000 ms isn't immediately obvious that it's 24 hours) if someone were to ever directly look at the constant.